### PR TITLE
Update logging attributes

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: Sources/HummingbirdWSCore/WebSocketInboundStream.swift [hunk 1]
```diff
@@ -78,7 +78,7 @@ public final class WebSocketInboundStream: AsyncSequence, Sendable {
                         return .init(from: frame)
                     default:
                         // if we receive a reserved opcode we should fail the connection
-                        self.handler.logger.trace("Received reserved opcode", metadata: ["opcode": .stringConvertible(frame.opcode)])
+                        self.handler.logger.debug("Received reserved opcode", metadata: ["hb.opcode": .stringConvertible(frame.opcode)])
```
Comment (Joannis) [inline Sources/HummingbirdWSCore/WebSocketInboundStream.swift:81]: Can we namespace this to websocket?

**Review by Joannis — CHANGES_REQUESTED** [review]

File: Sources/HummingbirdWSCore/WebSocketInboundStream.swift (hunk 1 above)
Comment (adam-fowler) [reply Sources/HummingbirdWSCore/WebSocketInboundStream.swift:81]: `hb.ws.opcode`?

Comment (adam-fowler) [reply Sources/HummingbirdWSCore/WebSocketInboundStream.swift:81]: or `[WebSocket] Received reserved opcode`?

Comment (Joannis) [reply Sources/HummingbirdWSCore/WebSocketInboundStream.swift:81]: `hb.ws.opcode` is what I was thinking

Comment (adam-fowler) [reply Sources/HummingbirdWSCore/WebSocketInboundStream.swift]: done

**Review by Joannis — APPROVED** [review]

**Pull request merged**